logic/src/commonMain/kotlin/com/wire/kalium/logic/data/team/TeamRepository.kt

```diff
@@ -139,7 +132,7 @@ internal class TeamDataSource(
                 teamId = teamId,
                 userId = userId,
             )
-        }.flatMap { _ ->
+        }.flatMap { _ -> // TODO jacob why dow we fetch team member info if we don't use the response?
```

Comment:
Yep, and it was changed by me 🫣

Indeed permission is the important part since it decides the `UserType` (standard, owner, admin, external). As it is now members added via `team.member-join`event will always get user type: `standard`.

I'll fix this